src/index.ts

```diff
@@ -903,7 +908,7 @@ async function prepareDepositTxs({
 	let depositTx: interfaces.IPeanutUnsignedTransaction
 	if (numberOfLinks == 1) {
 		try {
-			if (peanutContractVersion === 'v4.4') {
+			if (CA_SUPPORTS_MFA.includes(peanutContractVersion)) {
```

Comment:
this is wrong, "// Using the new, powerful and flexible deposit function!" is only supported in v4.4 and up. Vault contracts that support MFA are v4.3 and up